pymongoexplain/explainable_collection.py

```diff
@@ -149,29 +146,30 @@ def find_one_and_delete(self, filter: Document, projection: list = None,
                                        kwargs)
         return self._explain_command(command)
 
-    def find_one_and_replace(self, filter: Document, replacement: Document,
+    def find_one_and_replace(self, filter: Document, update:
```

Comment:
This argument should still be called "replacement", not "update". It needs to match Pymongo's find_one_and_replace

Reply:
Done.